### PR TITLE
Use taptic feedback on start button long press

### PR DESCRIPTION
**Pull request by juanlaube**
## What's this?
Adds taptic feedback when doing a long press on the start button to start a manual time entry.

### Relationships

Closes nothing

## Why do we want this?
:sogood:

## How is it done?
🐝 

### Why not another way?
🤷‍♂ 

### Side effects
Makes our app cooler

## Review hints
Please test on a device! I cannot test with iOS 13 right now 🙃 

## :squid: Permissions
🦑

### CODE REVIEW

**Comment by kornelijepetak** [comment]
taptic? Isn't it haptic? 😉

**Comment by juanlaube** [comment]
Taptic is correct (and weird) 😄  https://appleinsider.com/articles/16/09/27/inside-the-iphone-7-apples-taptic-engine-explained

**Comment by kornelijepetak** [comment]
> Taptic is correct (and weird) 😄 [appleinsider.com/articles/16/09/27/inside-the-iphone-7-apples-taptic-engine-explained](https://appleinsider.com/articles/16/09/27/inside-the-iphone-7-apples-taptic-engine-explained)

Oh, that's a product name. I thought you were referring to the concept of haptic feedback in general. 😄 It makes sense to call the product that does this with that name... 😄

**Comment by juanlaube** [comment]
Simon helped me to test this in Buenos Aires and we found it repeating the feedback after holding the long press for a while. We need to fix it to give the feedback only once.

File: Toggl.iOS/Extensions/Reactive/UIViewExtensions.cs [hunk 1]
```diff
@@ -44,6 +44,28 @@ public static IObservable<Unit> LongPress(this IReactive<UIView> reactive)
                 });
             });
 
+        public static IObservable<Unit> LongPressAndFeedback(this IReactive<UIView> reactive)
```
Comment (kornelijepetak) [inline Toggl.iOS/Extensions/Reactive/UIViewExtensions.cs]: Why a new method?

The only difference between this one and the LongPress is in the different gesture recognizer.
Why don't you instead parameterize `LongPress(this IReactive<UIView> reactive) into `LongPress(this IReactive<UIView> reactive, bool useFeedback).

And then: 

```cs
var gestureRecognizer = useFeedback
	? new UILongPressGestureRecognizer(() =>
        {
            var selectionFeedback = new UISelectionFeedbackGenerator();
            selectionFeedback.Prepare();
            selectionFeedback.SelectionChanged();
            observer.OnNext(Unit.Default);
        })
	: new UILongPressGestureRecognizer(() => observer.OnNext(Unit.Default));
```

The remainder of the method is exactly the same.
Dry it a bit. 😄

**Comment by amulware** [comment]
Why is this on hold? I'd like this either completed or closed.

**Comment by juanlaube** [comment]
This is ready for re-review, can be immediately merged after that.

**Pull request merged**